<launch>

    <!-- Node for naming the session ID -->
    <node name="talker" pkg="uav_pi" type="talker.py"/>

    <!-- Node for detecting objects -->
    <node name="object_detect_server" pkg="uav_pi" type="detect-pt.py"/>

    <!-- Node for processing detections -->
    <node name="detection_processor" pkg="uav_pi" type="detection_processor.py"/>

    <!-- Service for directly changing pwm signals to the Survey3 based on incoming commands-->
    <node name="survey3_control_service" pkg="uav_pi" type="survey3_control_service.py"/>

    <!-- Service for downloading and publishing pics from the survey3 -->
    <node name="survey3_publisher" pkg="uav_pi" type="survey3_publisher.py"/>

    <!-- Service for publishing temperature data from the MLX -->
    <node name="MLX_publisher" pkg="uav_pi" type="MLX_publisher.py"/>
</launch>
